modules/geochemistry/test/tests/interrogate_reactions/tests

```diff
@@ -2,6 +2,11 @@
   [./invalid_swap]
     type = RunException
     input = invalid_swap.i
+    # only run in opt mode, otherwise the Outputs/perf_graph=true in the CIVET debug tests triggers:
```

Comment:
This is something that we _really_ shouldn't be doing.

Reply:
I agree we shouldn't be doing this.  Do you know how to get around CIVET putting that Outputs/perf_graph=true in these tests?